ExceptionHandling.java

```diff
@@ -0,0 +1,24 @@
+public class ExceptionHandling {
+    int a,b,c;
```

Comment:
```suggestion
    int a,b,c;
```
Break this down into multiple statements.